content/docs/command-reference/index.md

```diff
@@ -15,8 +15,8 @@ does not change directories in your terminal).
 - Copy data files or dataset directories for modeling into the repository, and
   track them with DVC using the `dvc add` command.
 - Process the data with your own source code, using `dvc.yaml` and/or the
-  `dvc run` command, specifying further <abbr>outputs</abbr> that should also be
-  tracked by DVC after the code is executed.
+  `dvc stage add` command to specify further <abbr>outputs</abbr> that should
+  also be tracked by DVC, and executing the code using `dvc repro`.
```

Comment:
Maybe we should keep this simpler by not mentioning `stage add`. It's linked from the dvc.yaml guide anyway, and from what I remember editing that manually is the default recommended way to create stages.